iris_ugrid/tests/unit/ucube/test_UCube.py

```diff
@@ -0,0 +1,88 @@
+# Copyright Iris-ugrid contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Test basic :class:`iris_ugrid.ucube.Ucube` object.
+"""
+import iris.tests as tests
+
+import re
+
+from iris import Constraint
+from iris.tests import IrisTest, get_data_path
+
+from iris.cube import CubeList
```

Comment:
I don't understand this order of these imports.
Why isn't it:
```suggestion
from iris import Constraint
from iris.cube import CubeList
from iris.tests import IrisTest, get_data_path
```

Reply:
I'm just going to remove the `from iris.tests import` altogether, since [we're already importing `iris.tests` on L9](https://github.com/SciTools-incubator/iris-ugrid/pull/27/files/dc3665e5f25e89d2021a6f8129151e04bd85ee70#diff-ce33dc07f79a281ee7617e0e1270807764395b6022cc8ee8e15b04b719865a96R9).

Reply:
8cf8e942ac667a2a9be00fb8fdfcded2844d4fec